test/integration/client_test.go

```diff
@@ -34,18 +34,19 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/version"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/watch"
+	"github.com/GoogleCloudPlatform/kubernetes/test/integration/framework"
 )
 
 func init() {
-	requireEtcd()
+	framework.RequireEtcd()
```

Comment:
Suggestion: put this init function in the framework package, then you get it just by importing the package?